webapp/src/components/propertyValueElement.tsx

```diff
@@ -152,6 +154,7 @@ const PropertyValueElement = (props:Props): JSX.Element => {
             <EditableDayPicker
                 className='octo-propertyvalue'
                 value={value as string}
+                dateFormat={UserSettings.preferredDateFormat}
```

Comment:
I tiny problem here is this is not going to be reactive, not a huge deal, but probably something to fix later.